torch/_tensor.py

```diff
@@ -1319,6 +1319,32 @@ def to_sparse_coo(self):
         """
         return self.to_sparse()
 
+    def dim_order(self):
```

Comment:
You may want to add torch function override support here, look for things like `has_torch_function_unary` in this file